test/basics.jl

```diff
@@ -298,6 +298,90 @@ facts("Basic checks") do
         @fact value(d) --> 4*3
         @fact value(a) --> 4*3
         @fact value(b) --> 2*4*3
+
     end
-    
+
+    context("bindmap") do
+
+        src2dst(x) = x + 2
+        dst2src(x) = x - 2
+
+        # oneway with initiation
+        src = Signal(3)
+        dst = Signal(0)
+        bindmap!(dst, src2dst, src)
+        @fact value(dst) --> 5
+        push!(src, 2)
+        step()
+        @fact value(dst) --> 4
+        push!(dst, 5)
+        step()
+        @fact value(src) --> 2
+        unbind!(dst,src)
+        push!(src,1)
+        step()
+        @fact value(dst) --> 5
+
+        # twoway with initiation
+        src = Signal(3)
+        dst = Signal(0)
+        bindmap!(dst, src2dst, src, dst2src)
+        @fact value(dst) --> 5
+        push!(src, 2)
+        step()
+        @fact value(dst) --> 4
+        push!(dst, 5)
+        step()
+        @fact value(src) --> 3
+        unbind!(dst,src,false) # test oneway first
+        push!(src,1)
+        step()
+        @fact value(dst) --> 5
+        push!(dst,1)
+        step()
+        @fact value(src) --> -1
+
+        # This should work but I think line 444 in operators.jl shouldn't exit if the twoway unbind! was applied after a oneway unbind!.
+        # unbind!(dst,src) # test other way as well
+        # push!(dst,2)
+        # step()
+        # @fact value(src) --> -1
```

Comment:
Should this work now?

Reply:
I don't think so. [This line](https://github.com/JuliaGizmos/Reactive.jl/blob/5d1a8da7c6053330f2db31df6febdea028dfbe9c/src/operators.jl#L403), I think, only checks src=>dest but not dest=>src? so it just exists even if the other way around existed...